docker/alias/alias_computation_test.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Alias computation tests."""
+import datetime
+import os
+import unittest
+
+from google.cloud import ndb
+
+import osv
+from docker.alias import alias_computation
+from osv import tests
+
+TEST_DATA_DIR = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), 'testdata')
+
+
+class AliasTest(unittest.TestCase, tests.ExpectationTest(TEST_DATA_DIR)):
+  """Alias tests."""
+
+  def test_basic(self):
+    """Tests basic case."""
+    osv.AliasGroup(
+        bug_ids=['aaa-123', 'aaa-124'],
+        last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    alias_computation.main()
+    bug_ids = osv.AliasGroup.query(
+        osv.AliasGroup.bug_ids == 'aaa-123').get().bug_ids
+    self.assertEqual(['aaa-123', 'aaa-124'], bug_ids)
```

Comment:
Hmm, so do AliasGroups stay around even if none of the IDs are Bugs in the database?
I don't think we want that (though I don't know if it would happen in practice)

Reply:
Added Bugs to database.